graphql/types/ReturnRequest.graphql

```diff
@@ -187,6 +187,7 @@ input ReturnRequestFilters {
   sequenceNumber: String
   id: String
   createdIn: DateRangeInput
+  dateSubmitted: DateRangeInput
```

Comment:
Are you adding this field here just in order to use it in the middleware for the list API? If so, we can delete it from here and delete also in the args from the getRequestList.ts. Then, inside the returnRequestListService, we just use the `key === 'createdIn'` to create the filter.